enocean/__init__.py

```diff
@@ -205,7 +205,15 @@ def _process_packet_type_radio(self, data, optional):
             dBm = -optional[5]
             SecurityLevel = optional[6]
             self.logger.debug("enocean: radio message with additional info: subtelnum = {} / dest_id = {:08X} / signal = {}dBm / SecurityLevel = {}".format(subtelnum, dest_id, dBm, SecurityLevel))
-
+            if (choice == 0xD4) and (self.UTE_listen == True):
+                #self.logger.info("UTE FOUND!! RORG {:02x}".format(choice))
+                self.logger.info("call send_UTE_response")
+                self._send_UTE_response(data, optional)
+                #self.logger.debug("send response")
+                #logger.debug("time in milliseconds")
+                #logger.debug(datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3])
+                #self._send_radio_packet(5, choice, [0x91, 0xFF, 0x33, 0x00, 0x09, 0x01, 0xD2],[0x03, 0x01, 0x93, 0xAF, 0x59, 0xFF, 0x0] )#UTE response package for psc234 - 0193AF59
+                #self._send_radio_packet(8, choice, [0x91, 0x02, 0x46, 0x00, 0x12, 0x01, 0xD2],[0x03, 0x01, 0xA6, 0x20, 0x2B, 0xFF, 0x0] )#UTE response package for Nodon actor - 01A6202B
```

Comment:
Same here, what is this commented line for? Looks like you were trying something, which is fine but please remove for the pul request unless it is important.